src/main/webapp/script.js

```diff
@@ -10,11 +15,49 @@ function passQuery() {
         .then((data) => { 
           // data is a 2D array, where the first row is a header row and all
           // subsequent rows are one piece of data (e.g. for a state or county)
-          callOliviaFunction(data);
+          displayVisualization(data);
         });
       }
       else {
         console.log("There was an error");
       }
     });
-}
\ No newline at end of file
+}
+
+ // displayVisualization takes in a data array representing the 2D array returned by the 
+ // census API. The function creates the necessary visualization with the census data.
+function displayVisualization(censusDataArray) {
+  if (censusDataArray[0][2] == 'state' && censusDataArray[0][3] != 'county') {
+    google.charts.setOnLoadCallback(drawRegionsMap(censusDataArray));
```

Comment:
Please add some comments on what the expected format censusDataArray is

Reply:
fixed